lib/migration/src/Obelisk/Migration.hs

```diff
@@ -240,3 +240,8 @@ writeEdge dir actionFiles v1 v2 = do
       forM_ actionFiles $ \fp -> liftIO $
         writeFile (edgeDir </> T.unpack fp) ""
       pure $ Just edgeDir
+
+readFileStrict :: FilePath -> IO String
+readFileStrict = makeStrict <=< readFile
```

Comment:
[Text.IO.readFile](https://hackage.haskell.org/package/text-1.2.3.0/docs/Data-Text-IO.html#v:readFile) is strict and seems more appropriate here?

Reply:
Yep. I already had it checked out. Will fix.